src/test/java/org/apache/ibatis/type/ByteArrayUtilsTest.java

```diff
@@ -0,0 +1,39 @@
+/**
+ *    Copyright 2009-2019 the original author or authors.
```

Comment:
Is this copyright declaration correct? Seems a bit vague

Reply:
I dont really know much about these headers but this is what seems to be on the other classes (just with different dates)